.buildkite/others.rayci.yml

```diff
@@ -79,3 +79,12 @@ steps:
         --test-env=USERPROFILE
     depends_on: windowsbuild
     soft_fail: true
+
+  # bot
+  - label: ":robot_face: CI state machine bot"
+    tags: skip-on-premerge
+    if: build.branch == "master" && build.env("RAYCI_CONTINUOUS_BUILD") == "1"
```

Comment:
why is `RAYCI_CONTINUOUS_BUILD` set to 1? on nighly?

Reply:
I'll create a schedule run on postmerge with that env set: https://buildkite.com/ray-project/postmerge/settings/schedules. It's not there yet.